packages/expo-gl/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### 🎉 New features
 
+- [gl] Delete `prop-types` in favor of TypeScript. ([#8675](https://github.com/expo/expo/pull/8675) by [@EvanBacon](https://github.com/EvanBacon))
```

Comment:
Hmm probably we will need another section in changelog for "other changes" – this is neither a fix nor a new feature 😅 Anyway let's leave it as is for now as I'll need to make some changes in expotools for that.